gradle/plugins/build.gradle.kts

```diff
@@ -24,9 +24,6 @@ dependencies {
     implementation(libs.detekt.gradle.plugin) {
         exclude("io.github.detekt.sarif4k", "sarif4k")
     }
-    implementation(libs.diktat.gradle.plugin) {
-        exclude("io.github.detekt.sarif4k", "sarif4k")
```

Comment:
strange exclusion for our plugin, but still -- suggestion to comment for now instead of removing

Reply:
yes